api/handler/v2/tx.go

```diff
@@ -52,6 +57,12 @@ func (h *Handler) BroadcastTx(ctx *fasthttp.RequestCtx) {
 		return
 	}
 
+	err = ValidateBroadcastTxParams(&reqParams)
+	if err != nil {
+		sendBadRequestResponse(ctx, err.Error())
```

Comment:
log error

Reply:
I am trying to avoid noise by just logging internal errors since we dont have log levels yet. This are user input errors to which the error response will have the message of what was missing in the request. Will implement log levels in a following PR and then can add the log.info / log.debug in this scenarios if thats ok with you